app.json

```
@@ -0,0 +1,28 @@
+{
+    "name": "bolt-js-starter-template",
+    "description": "A Slack Bolt JS starter app deployed via Heroku",
+    "repository": "https://github.com/allhaile/bolt-js-starter-template",
+    "logo": "https://docs.slack.dev/img/logos/slack-developers-white.png",
+    "keywords": ["node", "slack", "bolt", "javascript", "starter"],
+    "env": {
+      "SLACK_BOT_TOKEN": {
+        "description": "Bot User OAuth Access Token",
```

suggestion: I think we could call this "Slack Bot Token" to keep it short and simple like the others:
```suggestion
        "description": "Slack Bot Token",
```